app/src/main/java/com/github/campus_capture/bootcamp/fragments/MapsFragment.java

```diff
@@ -40,7 +95,8 @@ public class MapsFragment extends Fragment{
 
         //Build and initialize the DB
         //The DB contains the Zone around the campus
-        ZoneDatabase zoneDB = Room.databaseBuilder(getActivity(),
+
+        zoneDB = Room.databaseBuilder(getActivity(),
                 ZoneDatabase.class, "zones-db")
                 .createFromAsset("databases/zones-db.db")
                 .allowMainThreadQueries().build();
```

Comment:
We will need to change the `.allowMainThreadQueries()` as it might slow down the UI if queries takes more time. But that's on my PR